src/app/shared/models/classes/coffee.class.ts

```diff
@@ -17,7 +17,7 @@ export class Coffee extends Product {
             }
 
             if (coffeeShape.tastingNotes != null) {
-                this.tastingNotes = coffeeShape.tastingNotes;
+                this.tastingNotes = (coffeeShape.tastingNotes as any as string).split(", ");
```

Comment:
Same reasoning as with [this comment](https://github.com/lpboucher/dak-coffee-wholesale/pull/92#discussion_r677268023) on the [Flavour-filters PR](https://github.com/lpboucher/dak-coffee-wholesale/pull/92).